OrbitGl/App.cpp

```diff
@@ -677,8 +684,15 @@ outcome::result<void, std::string> OrbitApp::OnLoadSession(
 
 //-----------------------------------------------------------------------------
 void OrbitApp::LoadSession(const std::shared_ptr<Session>& session) {
-  if (SelectProcess(Path::GetFileName(session->m_ProcessFullPath))) {
+  if (Capture::GTargetProcess->GetFullPath() == session->m_ProcessFullPath) {
+    GOrbitApp->LoadModulesFromSession(Capture::GTargetProcess, session);
+    return;
+  } else if (SelectProcess(Path::GetFileName(session->m_ProcessFullPath))) {
     Capture::GSessionPresets = session;
```

Comment:
check for error first move is out for if-else-if-else

Reply:
done